screens/DetailScreen.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Image, StyleSheet, Text, View } from 'react-native';
+
+export default function DetailScreen({  route }) {
+  const { title, edition_count, authors } = route.params;
+
+  const author = authors.map((author) => {
+    return author.name
+
+  });
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>{title}</Text>
+      <Text style={styles.subtitle}>{author.length > 1 ? `Authors: ${author}, ` : `Author: ${author}`}</Text>
```

Comment:
I haven't jet figured out a nice way to identify last item of authors array which should not have a comma . 
Maybe something like :  
`{author.length > 1 || === author.length -1  ? Authors: ${author},  : Author: ${author}}`

Reply:
1. think if there is a better name for the `author` variable
2. `author.length == author.length -1` will always be false
2. why does it even work as it is at the moment?
3. take a look at [Array.prototype.join()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)

Reply:
thanks , I don't know why I did that.. fixed in f4239d901e05b42a1e30b540408b9b09ab87650d

1.  author is now listOfAuthors
2. indeed non sense idea 
3.  I guess it works because my params is authors, so author works but it dosen't make sense logically to be called like that 
4.  very useful

Reply:
about 3.: i was more thinking of you implicitly using the [Array.prototype.toString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toString) method